### PR TITLE
Post reference data results

### PR DESCRIPTION
**Pull request by epassaro**
## Description
Several changes made to `compare-refdata` pipeline:

- Pipeline is run by commenting `/azp run compare-refdata` on PR
- Results are stored in `OpenSupernova.org` by PR number and commit
- Results (failed or passing) are posted through comments by Azure Pipelines bots (see below)

## Motivation and Context
Part of the CI project

## How Has This Been Tested?
- [x] Testing pipeline
- [x] Other (please describe)

`compare-refdata` pipeline

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] None of the above (please describe)

## Checklist:

- [x] My code follows the code style of this project
- [x] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly
- [ ] I have built the documentation on my fork following [these instructions](https://tardis-sn.github.io/tardis/development/documentation_preview.html)
- [x] I have assigned and requested two reviewers for this pull request

### CODE REVIEW

File: azure-pipelines/compare-refdata.yml [hunk 1]
```diff
@@ -13,26 +13,45 @@ schedules:
...
-          $(package.manager) install bokeh --channel conda-forge --no-update-deps --yes
-        displayName: 'Install Bokeh'
+          GIT_LFS_SKIP_SMUDGE=1 git clone https://github.com/tardis-sn/tardis-refdata.git $(refdata.dir)
+          cd $(refdata.dir)
+          git lfs pull --include="atom_data/kurucz_cd23_chianti_H_He.h5"
+          git lfs pull --include="atom_data/chianti_He.h5"
+          git lfs pull --include="unit_test_data.h5"
+          git lfs pull --include="packet_unittest.h5"
+          git lfs pull --include="montecarlo_1e5_compare_data.h5"
+          git lfs pull --include="montecarlo_one_packet_compare_data.h5"
```
Comment (andrewfullard) [inline azure-pipelines/compare-refdata.yml:49]: Slightly concerned about LFS usage here

**Review by andrewfullard — APPROVED** [review]
Needs documentation ASAP!

**Comment by epassaro** [comment]
> Needs documentation ASAP!

Basic usage is [already documented](https://tardis-sn.github.io/tardis/development/continuous_integration.html#reference-data-pipeline), could merge this and move on to next steps?

 - Compare reference data for any two arbitrary commits (currently we are just comparing `HEAD` of PR against `upstream/master`)
 - Update reference data from pipelines
 - Then, update documentation for good

**Pull request merged**